content/4_Gold/Intro_DP.mdx

```diff
@@ -132,11 +128,51 @@ It's usually a good idea to write a slower solution first. For example, if the c
 
 </Info>
 
-<IncompleteSection>
+**Dynamic Programming** (DP) is an important algorithmic technique in Competitive Programming from the gold division to competitions like the International Olympiad of Informatics. By breaking down the full task into sub-problems, DP avoids the redundant computations of brute force solutions.
+
+Although it is not too difficult to grasp the general ideas behind DP, the technique can be used in a diverse range of problems and is a must-know idea for competitors in the USACO Gold division.
+
+Often, dynamic programming can be split into 4 steps: identify the state, the recurrence, the base case, and the answer. 
+
+### State
+
+The state of our dynamic programming is what each spot in our dp array/matrix represents. We want to set up the states such that the answer is one of the states, and there is a clear transition between states.
+
+One way to think about states is to ask yourself "what information do I need to know to identify my state / progress in solving this problem?" For example, the state for fibonacci could be i, representing that you're trying to find the i'th fibonacci number.
```

Comment:
Use LaTeX for "i". Also, keep the punctuation and spacing consistent and correct